azure-pipelines.sds.pr-release.yml

```diff
@@ -0,0 +1,402 @@
+              kvSqlPasswordSecret: ${{ variables.vhSqlPasswordSecret }}
+              kvSqlUsernameSecret: ${{ variables.vhSqlUsernameSecret }}
+
+#####################################################
+# Deploy Helm Chart to Dev. ######################### 
+  - stage: Install
+    dependsOn: Run_Entity_Framework_Dev
+    displayName: Deploy Helm Chart Dev AKS
+    variables:
+      - template: variables/dev.yaml
+      - template: variables/shared.yaml
+        parameters:
+          env: ${{ variables.env }}
+      - name: dnsRecordName
+        value: vh-${{ variables.git_repo_name }}-${{ variables.prTagPrefix }}
+    jobs:
+      - job: helm_uninstall
+        displayName: Uninstall Helm from Dev
+        steps:
+
+          - template: templates\Azure\Aks\get-active-cluster.yaml@azTemplates
+            parameters:
+              subscriptionName: "${{ variables.subscriptionName }}"
+              environment: ${{ variables.env }}
+            
+          - template: templates/Containerisation/helm/helm-uninstall.yaml@azTemplates
+            parameters:
+              namespace: 'vh'
+              releaseName: "${{ variables.chartName }}-${{ variables.prTagPrefix }}"
+              azureSubscription: "${{ variables.subscriptionName }}"
+              aksResourceGroupName: "$(aksCluster.resourceGroup)"
+              aksClusterName: "$(aksCluster.name)"
+      - job: aks_push
+        displayName: Deploy To Dev AKS
+        steps:
+
+          - template: templates\Azure\Aks\get-active-cluster.yaml@azTemplates
+            parameters:
+              subscriptionName: "${{ variables.subscriptionName }}"
+              environment: ${{ variables.env }}
+          - template: templates/Containerisation/helm/helm-install.yaml@azTemplates
+            parameters:
+              privateKeyFileName: ${{ variables.gh_app_private_key_file_name }}
+              appId: $(gh_app_app_id)
+              envValuesTemplatePath: ${{ variables.envValuesTemplatePath }}
+              envValuesPath: ${{ variables.envValuesPath }}
+              acrName: ${{ variables.acrName }}
+              repositoryName: ${{ variables.repositoryName }}
+              prTagName: ${{ variables.prTagName }}
+              releaseName: "${{ variables.chartName }}-${{ variables.prTagPrefix }}"
+              dnsRecordName: ${{ variables.dnsRecordName }}
+              dnsZone: ${{ variables.dnsZone }}
+              env: ${{ variables.env }}
+              chartName: ${{ variables.chartName }}
+              chartPath: ${{ variables.chartPath }}
+              subscriptionName: "${{ variables.subscriptionName }}"
+              aksResourceGroupName: "$(aksCluster.resourceGroup)"
+              aksClusterName: "$(aksCluster.name)"
+              dnsSubscriptionName: "${{ variables.dnsZoneSubscription }}"
+              dnsResourceGroupName: "${{ variables.dnsZoneResourceGroup }}"
+
+      - ${{ if eq(variables.run_acceptance_tests, true) }}:
+        - job: AcceptanceTestsDev
```

Comment:
Have you added all the acceptance test settings from here
https://github.com/hmcts/vh-notification-api/blob/71aa3a9a0b179fca6da15fe222b80ebe3a6e4639/azure-pipelines.yml#L42